remoteappmanager/handlers/base_handler.py

```diff
@@ -17,8 +28,35 @@ def render(self, template_name, **kwargs):
         args = dict(
             user=self.current_user,
             base_url=self.application.config.base_url,
-            logout_url=urljoin(self.application.config.hub_prefix, "logout")
-        )
+            logout_url=urljoin(self.application.config.hub_prefix, "logout"))
 
         args.update(kwargs)
         super(BaseHandler, self).render(template_name, **args)
+
+    def verify_token(self, cookie_name, encrypted_cookie):
```

Comment:
My only concern is that this code is not asyncronous, meaning that we assume the request operation to be quick. This code is from jupyterhub, and they do the same, but it's a potential concern for us.

I am going to merge this, but we might have to rework this routine with a threadpool (like in async docker client) and issue the request asynchronously.